supervision/utils/internal.py

```diff
@@ -141,3 +142,42 @@ def __get__(self, owner_self: object, owner_cls: type) -> object:
         The result of calling the function stored in 'fget' with 'owner_cls'.
         """
         return self.fget(owner_cls)
+
+
+def get_instance_variables(cls: Any, include_properties=False) -> Set[str]:
```

Comment:
Add `include_protected` and maaaaaaybe `include_private` arguments.

Reply:
Okay, I've just learned something. I knew that prepending `__` scopes the var with the class name. I expected to see `__MockClass__private_var`.

No! It's actually `_MockClass__private_var`! With a single underscore prefix!

I won't add neither `include_protected` nor `include_private`, as it adds a bit more complication than I like. If anyone needs it, they can add it themselves, and now there will be mock classes in the tests that will reveal the issue.